modules/virt-about-resource-quota-limits.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-resource-quota-limits_{context}"]
+= About resource quota limits
+
+You can only set `ResourceQuotas` for virtual machines if the resource quota has a set of requests. If the resource quota contains both requests and limits, the request limits on the virtual machine must be set.
```

Comment:
> If the resource quota contains both requests and limits, the request limits on the virtual machine must be set.
This sentence sounds confusing for me.. 

I think there should be AND:
` the requests AND limits on the virtual machine must be set.`

Reply:
or:
`the resource limits on the virtual machine must be set.`

Reply:
Well spotted. I think that was actually supposed to say "resource limits".

Reply:
Thanks @dshchedr . Updated.

Reply:
Well spotted @dshchedr, I believe this should have said "resource limits"

Reply:
+1, thanks to you both.